### ISSUE TITLE
sayit doesn't start after scheduled restart since upgrade to JS-Controller 3.1.6

### ISSUE THREAD

**Issue by Qlink84** [ioBroker/ioBroker.sayit#104]
sayit doesn't start after scheduled restart since upgrade to JS-Controller 3.1.6
happens to all my sayit instances...
had no problems with this settings on JS-Controller 1.5.14

```
2020-08-29 03:15:01.244 - info: sayit.0 (7592) Scheduled restart.
--
2020-08-29 03:15:01.244 - info: sayit.0 (7592) stopping...
2020-08-29 03:15:01.759 - info: sayit.0 (7592) Terminated (ADAPTER_REQUESTED_TERMINATION): Without reason
2020-08-29 03:15:01.759 - info: sayit.0 (7592) terminating
2020-08-29 03:15:02.290 - error: host.iob01 instance system.adapter.sayit.0 terminated by request of the instance itself and will not be restarted, before user restarts it.
2020-08-29 03:15:02.290 - info: host.iob01 Do not restart adapter system.adapter.sayit.0 because desired by instance
```

sysinfo:

Windows Server 2019 (Hyper-V VM Gen2)
sayit: 1.9.8 (3 instances)
Node.js: 10.17.0
NPM: 6.11.3
js-controller: 3.1.6

**Comment by Apollon77** [comment]
Please check 1.11.4

**Comment by Qlink84** [comment]
i've installed 1.11.4 today with "npm install iobroker.sayit@1.11.4 --production" but adapter didn't start after installation.

log showed this:
```
sayit.0 | 2020-09-24 07:36:20.340 | info | (7940) stopping...
-- | -- | -- | --
sayit.0 | 2020-09-24 07:36:19.825 | info | (7940) Terminated (NO_ERROR): Without reason
sayit.0 | 2020-09-24 07:36:19.825 | info | (7940) terminating
sayit.0 | 2020-09-24 07:36:19.309 | error | (7940) Error in system.adapter.sayit.0: The unload method must return a Promise if it does not accept a callback!
sayit.0 | 2020-09-24 07:36:19.309 | info | (7940) stopping...
sayit.0 | 2020-09-24 07:36:19.309 | info | (7940) All files uploaded
sayit.0 | 2020-09-24 07:36:19.294 | info | (7940) Upload announce mp3 files
sayit.0 | 2020-09-24 07:36:19.294 | info | (7940) Install process. Upload files and stop.
sayit.0 | 2020-09-24 07:36:19.247 | info | (7940) starting. Version 1.11.4 in C:/iobroker/iob01/node_modules/iobroker.sayit, node: v10.17.0, js-controller: 3.1.6

```
i did an "upload" of sayit adapter through admin ui and now adapter seems like running fine now.
i'll come back to you if the restart problem is fixed now.


```
sayit.1 | 2020-09-24 07:37:37.021 | info | (7416) All files uploaded
-- | -- | -- | --
sayit.1 | 2020-09-24 07:37:36.990 | info | (7416) Upload announce mp3 files
sayit.1 | 2020-09-24 07:37:36.927 | info | (7416) starting. Version 1.11.4 in C:/iobroker/iob01/node_modules/iobroker.sayit, node: v10.17.0, js-controller: 3.1.6
sayit.0 | 2020-09-24 07:37:36.911 | info | (5680) All files uploaded
sayit.2 | 2020-09-24 07:37:36.912 | info | (7664) All files uploaded
sayit.0 | 2020-09-24 07:37:36.880 | info | (5680) Upload announce mp3 files
sayit.2 | 2020-09-24 07:37:36.849 | info | (7664) Upload announce mp3 files
sayit.2 | 2020-09-24 07:37:36.787 | info | (7664) starting. Version 1.11.4 in C:/iobroker/iob01/node_modules/iobroker.sayit, node: v10.17.0, js-controller: 3.1.6
sayit.0 | 2020-09-24 07:37:36.786 | info | (5680) starting. Version 1.11.4 in C:/iobroker/iob01/node_modules/iobroker.sayit, node: v10.17.0, js-controller: 3.1.6
host.iob01 | 2020-09-24 07:37:36.504 | info | iobroker exit 0
host.iob01 | 2020-09-24 07:37:35.489 | info | instance system.adapter.sayit.2 started with pid 7664
host.iob01 | 2020-09-24 07:37:35.489 | info | instance system.adapter.sayit.1 started with pid 7416
host.iob01 | 2020-09-24 07:37:35.473 | info | instance system.adapter.sayit.0 started with pid 5680
host.iob01 | 2020-09-24 07:37:35.395 | info | iobroker Update "system.adapter.sayit.2"
host.iob01 | 2020-09-24 07:37:35.395 | info | iobroker Update "system.adapter.sayit.1"
host.iob01 | 2020-09-24 07:37:35.395 | info | iobroker Update "system.adapter.sayit.0"
host.iob01 | 2020-09-24 07:37:35.301 | info | iobroker upload [0] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/blockly.js blockly.js application/javascript
host.iob01 | 2020-09-24 07:37:35.239 | info | iobroker upload [1] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/engines.js engines.js application/javascript
host.iob01 | 2020-09-24 07:37:35.176 | info | iobroker upload [2] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/de/translations.json i18n/de/translations.json application/json
host.iob01 | 2020-09-24 07:37:35.114 | info | iobroker upload [3] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/en/translations.json i18n/en/translations.json application/json
host.iob01 | 2020-09-24 07:37:35.051 | info | iobroker upload [4] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/es/translations.json i18n/es/translations.json application/json
host.iob01 | 2020-09-24 07:37:34.989 | info | iobroker upload [5] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/fr/translations.json i18n/fr/translations.json application/json
host.iob01 | 2020-09-24 07:37:34.926 | info | iobroker upload [6] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/it/translations.json i18n/it/translations.json application/json
host.iob01 | 2020-09-24 07:37:34.864 | info | iobroker upload [7] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/nl/translations.json i18n/nl/translations.json application/json
host.iob01 | 2020-09-24 07:37:34.801 | info | iobroker upload [8] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/pl/translations.json i18n/pl/translations.json application/json
host.iob01 | 2020-09-24 07:37:34.739 | info | iobroker upload [9] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/pt/translations.json i18n/pt/translations.json application/json
host.iob01 | 2020-09-24 07:37:34.676 | info | iobroker upload [10] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/ru/translations.json i18n/ru/translations.json application/json
host.iob01 | 2020-09-24 07:37:34.612 | info | iobroker upload [11] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/i18n/zh-cn/translations.json i18n/zh-cn/translations.json application/json
host.iob01 | 2020-09-24 07:37:34.549 | info | iobroker upload [12] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/index.html index.html text/html
host.iob01 | 2020-09-24 07:37:34.487 | info | iobroker upload [13] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/index_m.html index_m.html text/html
host.iob01 | 2020-09-24 07:37:34.424 | info | iobroker upload [14] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/sayit.png sayit.png image/png
host.iob01 | 2020-09-24 07:37:34.346 | info | iobroker upload [15] sayit.admin C:/iobroker/iob01/node_modules/iobroker.sayit/admin/words.js words.js application/javascript
host.iob01 | 2020-09-24 07:37:33.284 | info | iobroker upload sayit
```

**Comment by Apollon77** [comment]
sorry, my fault ... fixed in 1.11.5
